<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>436</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Book Antiqua</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDIT BOOK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="description_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="id_input">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="page_count_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="isbn_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="yes_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="no_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="year_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>134</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ISBN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page Count</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Issued</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
